### DISCUSSION TITLE
Storing the logs of the grafana agent server

### DISCUSSION THREAD

**Discussion by rosh0450** [grafana-cold-storage/agent#6352] [General]
How to define the path of the directory to store grafana agent server logs.
By default, it is getting stored in "/var/log/messages".
couldn't find it here: https://grafana.com/docs/agent/latest/static/configuration/server-config/

**Comment by tpaschalis** [comment]
Grafana Agent outputs all its logs on stderr. If you're installing Grafana Agent through our Linux packages, the log location is defined by your journald (or equivalent) setup. If you're running Grafana Agent as a binary, you can redirect its logs to whichever file you choose.
